lib/synsbasen_api.rb

```diff
@@ -9,6 +9,7 @@
 require "synsbasen_api/resources/vehicle"
 require "synsbasen_api/resources/variant"
 require "synsbasen_api/resources/version"
+require "ostruct"
```

Comment:
👁️

Reply:
When I removed Faraday from here I had to require the OpenStruct library manually 🤷‍♂️